tests/flow/test_data/onnx_string_identity.py

```diff
@@ -1,29 +1,6 @@
 import onnx
 from onnx import helper, TensorProto, ModelProto, defs
 
-
-def make_model(graph, **kwargs):  # type: (GraphProto, **Any) -> ModelProto
```

Comment:
let's validate ONNX versions vs ONNXRuntime versions here please

Reply:
Validated - using ONNX 1.8.0 when running this script will create a model that runs properly on ONNXRuntime 1.8.1.
Note that if we use ONNX latest (1.10.1), Opset version is 15, while ONNXRuntime 1.8.1 supports up to 14. I assume that ONNXRumtime will support Opset version 15...